md/deploy-bundle.md

```diff
@@ -23,10 +23,89 @@ It is useful to update the configuration, locally or from a remote computer.
-2. Unzip the `deploy.zip` into a temporary folder.
-3. Install Bonita BPM in your Application Server, by merging the appropriate folder (e.g. `Wildfly-10.1.0.Final` or `Tomcat-7.0.67`) from `BonitaBPMSubscription-x.y.z-deploy` folder
-into the existing root folder of your application server.
-4. Follow the [Tomcat bundle steps](tomcat-bundle.md#configuration) or [WildFly bundle steps](wildfly-bundle.md#configuration)
+2. Unzip the `deploy.zip` into a temporary folder (DEPLOY_ZIP_HOME).
+4. Follow the [Tomcat bundle steps](#tomcat-installation) or [WildFly bundle steps](#wildfly-installation)
+
+<a id="tomcat-installation" />
+
+## Tomcat installation
+### Install Bonita BPM Platform in Tomcat
+
+Copy all files and directories from DEPLOY_ZIP_HOME/Tomcat-7.0.76/server to your Tomcat root directory (TOMCAT_HOME).
+:::warning
+Some configurations files from the deploy zip will overwrite some default tomcat configuration files. Proceed
+with care in a tomcat where other applications are already installed.
+:::
+
+### Configure data sources
+
+Configuration of data source for Tomcat is split in two parts: because Tomcat doesn't support JTA natively,
+one data source will be configured in the Bitronix configuration file and the other data source will be configured
+in the standard Tomcat context configuration file.
+
+Non-transactional data source
+1. Open the file TOMCAT_HOME/conf/Catalina/localhost/bonita.xml
+2. Remove or comment out the lines regarding the h2 database.
+3. Uncomment the line matching your RDBMS.
+4. Update following attributes value:
+    - username: your RDBMS user name.
+    - password: your RDBMS password.
+    - url: the URL, including the RDBMS server address, RDBMS server port and database (schema) name.
+5. Repeat operations 2. to 4. for Business Data data source
+
+JTA data source (managed by Bitronix)
+1. Open the file TOMCAT_HOME/conf/bitronix-resources.properties
+2. Remove or comment out the lines regarding the h2 database.
+3. Uncomment the line matching your RDBMS and edit values for each settings (resource.ds1.*) according to your database installation.
+5. Repeat operations 2. and 3. for Business Data data source (resources.ds2.*)
+
+### Configure RDBMS vendor
+
+1. Open the file TOMCAT_HOME/bin/setenv.(bat|sh)
+2. Change the value of sysprop.bonita.db.vendor according to your RDBMS vendor
+3. Change the value of sysprop.bonita.bdm.db.vendor according to your RDBMS vendor
+
+### Add Jdbc driver
+You need to add your jdbc driver in TOMCAT_HOME/server/lib. 
+MySQL and PostgreSQL drivers can be found in deploy bundle under DEPLOY_ZIP_HOME/setup/lib directory. For other RDMS, 
+use the driver provided by your RDBMS vendor
+
+<a id="wildfly-installation" />
+
+## Wildfly installation
+
+### Install Bonita BPM Platform in Wildfly
+Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final to your Wildfly root directory (WILDFLY_HOME).
```

Comment:
This is not consistent with the instructions to install the bundle in Tomcat:

Tomcat installation:
_"Copy all files and directories from DEPLOY_ZIP_HOME/Tomcat-7.0.76/**server**" to your Tomcat root directory (TOMCAT_HOME)."_

Wildfly installation:
_"Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final to your Wildfly root directory (WILDFLY_HOME)."_

Which one is the good one?